composables/useContactForm.ts

```diff
@@ -15,7 +15,7 @@ export default function useContactForm (
   const email = ref(prefill.value?.email || '')
   const subject = ref(prefill.value?.subject || '')
   const message = ref(prefill.value?.message || '')
-  const bank = ref(prefill.value?.bank || '')
+  const bank = ref(prefill.value?.bank || null)
```

Comment:
null is important cuz this threw an error otherwise (already before my changes) since `modelValue` for `BankSearch` can only be either an `Object` or `null` and not a `string`